### DISCUSSION TITLE
Can i set sandboxieplus so that if i have a virus inside a sandboxie that virus can't read text documents inside a folder?

### DISCUSSION THREAD

**Discussion by Kevin5789** [sandboxie-plus/Sandboxie#4143] [General]
Can i set sandboxieplus so that if i have a virus inside a sandboxie that virus can't read text documents inside a folder?

**Comment by hg421** [comment]
Yes, if you set that folder as a [`ClosedFilePath`](https://sandboxie-plus.com/sandboxie/closedfilepath/) (blocks all access) or [`WriteFilePath`](https://sandboxie-plus.com/sandboxie/writefilepath/) (hides existing files outside the sandbox, but allows creating new files).
In the Sandboxie-Plus UI these correspond the the access types "Closed" or "Box Only".
![grafik](https://github.com/user-attachments/assets/39a0faea-06f1-4c1b-a11c-40e6dcc202bf)

You could set the box type to one of the "Privacy Protection" ones (requires a supporter certificate), which will hide personal files by default, only allowing read access to certain whitelisted locations.

**Comment by Kevin5789** [comment]
If when i create a new sandbox i delete a text document inside that sandbox (only in the sandbox not in the normal folder), then if i get a virus inside that sandbox the virus can read that text document that i deleted?

**Reply by bastik-1001** [reply]
No, not if I understand what you meant, correctly. A file inside a sandbox is removed, if it is deleted in the sandbox. If it was a copy and the source can't be read, due to access rules, the file can't be read. By default, Sandboxie does not allow raw disc access, so a deleted file can't be recovered.

**Reply by Kevin5789** [reply]
ok thank you
